src/cascadia/LocalTests_TerminalApp/CommandlineTest.cpp

```diff
@@ -971,21 +971,69 @@ namespace TerminalAppLocalTests
                 L"Pass a flag that would be accepted by split-pane, but isn't accepted by new-tab"));
             AppCommandlineArgs appArgs{};
             std::vector<const wchar_t*> rawCommands{ L"wt.exe", L"slpit-pane", L"-H" };

```

Comment:
Is "slpit-pane" an intentional typo in this whole file?

Reply:
You’re reading the TypoTests so yes

Reply:
Derp ok.